DESCRIPTION

```diff
@@ -22,12 +22,12 @@ Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.2.3
 Imports: 
-    httr,
+    httr2,
     jsonlite
 URL: https://github.com/jonocarroll/ntfy
 BugReports: https://github.com/jonocarroll/ntfy/issues
 Depends: 
-    R (>= 2.10)
+    R (>= 4.1)
```

Comment:
I've been hesitant to do this in the past, but that version is 2021, so - people should _probably_ upgrade.